packages/eds-core-react/src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -0,0 +1,68 @@
+import { forwardRef, HTMLAttributes } from 'react'
+import styled, { css } from 'styled-components'
+import { group as tokens } from './ButtonGroup.tokens'
+
+const { border } = tokens
+
+export type ButtonGroupProps = {
+  /** Display ButtonGroup vertically. */
+  vertical?: boolean
+} & HTMLAttributes<HTMLDivElement>
+
+const radius = border.type === 'border' && border.radius
+
+const ButtonGroupBase = styled.div<ButtonGroupProps>`
+  > * {
+    border-radius: 0;
+    @media (hover: hover) and (pointer: fine) {
+      &:hover {
+        border-radius: 0;
+      }
+    }
+  }
+  ${({ vertical }) =>
+    vertical
+      ? css`
+          display: grid;
```

Comment:
Can we change it to always be display flex, and change `flex-direction` from` row` to` column` when verical?

Reply:
done

Reply:
I tested some more without the "stack" wrapper, and realized it needs to be display: inline-flex actually (mui does the same) otherwise it will stretch 🤪

Reply:
done